### PR TITLE
App drawer requests

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Changes: 
1. Added 3D PCA example to the scatter plot cards. See here: http://localhost:3000/?appcard=scatterplot
2. Enabled citation button on the example level for the PNET data matrix example. See here: http://localhost:3000/?appcard=dataMatrix. 
3. Added description to the gene expression card. See from the main page: http://localhost:3000/

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: front/public/cards/scatterplot.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "ribbonMessage": "Please see our newest examples below: Continuous Variables and 3D PCA.",
     "ppcalls": [
         {
             "label": "Tumor methylome tSNE",
```
Comment (xzhou82) [inline front/public/cards/scatterplot.json:5]: cite PNET paper

File: front/public/cards/scatterplot.json [hunk 2]
```diff
@@ -61,6 +76,44 @@
                     "svg": 1
                 }
             }
+        },
+        {
+            "label": "3D PCA",
```
Comment (xzhou82) [inline front/public/cards/scatterplot.json:81]: cite survivor

File: front/public/cards/scatterplot.json [hunk 3]
```diff
@@ -46,10 +47,24 @@
                         "plots": [
                             {
                                 "chartType": "summary",
```
Comment (xzhou82) [inline front/public/cards/scatterplot.json:49]: cite survivor

File: front/public/cards/scatterplot.json (hunk 1 above)
Comment (xzhou82) [reply front/public/cards/scatterplot.json:5]: rather than showing a CITE button, can it show `Data Source` instead?

Comment (creilly8) [reply front/public/cards/scatterplot.json:5]: Yup, updated in [29dad8a](https://github.com/stjude/proteinpaint/pull/1865/commits/29dad8a3844a76f6a007c2db72b0d06993e5c27e).

**Review by xzhou82 — APPROVED** [review]
thanks

**Pull request merged**